recipes/notify.rb

```diff
@@ -68,13 +68,21 @@
       require 'uri'
 
       begin
-        req = Net::HTTP::Post.new('/events/')
+        ENV['GRAPHITE_PORT'] == '443' ? graphite_protocol = 'https' : graphite_protocol = 'http'
+        graphite_host = ENV['GRAPHITE_HOST'] || 'graphite.cluster'
```

Comment:
`GRAPHITE_HOST` is always set and defaults to `graphite.cluster` (https://github.com/alphagov/govuk-puppet/blob/f6f47118910f095909b4c2018c41b6aed516693d/modules/govuk_jenkins/manifests/jobs/deploy_app.pp) so I'm not sure you need the `||` here?